src/foundry/client/pixi/layers/controls/ruler.d.mts

```diff
@@ -238,10 +339,27 @@ declare global {
      */
     protected _postMove(token: Token): Promise<void>;
 
+    /**
+     * Broadcast Ruler measurement if its User is the connected client.
+     * The broadcast is throttled to 100ms.
+     */
+    protected _broadcastMeasurement(): void;
+
+    /**
+     * Package Ruler data to an object which can be serialized to a string.
+     */
+    protected _getMeasurementData(): Ruler.MeasurementData;
+
+    /**
+     * Update a Ruler instance using data provided through the cursor activity socket
+     * @param data - Ruler data with which to update the display
+     */
+    update(data: Ruler.MeasurementData | null): void;
+
     /**
      * Handle the beginning of a new Ruler measurement workflow
      * @param event - The drag start event
-     * @see Canvas._onDragLeftStart
+     * @see Canvas.#onDragLeftStart
```

Comment:
Is `Canvas.#onDragLeftStart` referring to an truly private property named `#onDragLeftStart` or is it trying to refer to an instance side property?

Reply:
Private property, they made some more stuff private this cycle